code/modules/customitems/item_defines.dm

```diff
@@ -146,6 +146,38 @@
 	icon_state = "kidosvest"
 	item_state = "kidosvest"
 
+//Kluys fluff nanofibre suit
+/obj/item/clothing/suit/kluysfluff
+	name = "Nano Fibre Jacket"
+	desc = "A Black Suit made out of nanofibre. The newest of cyberpunk fashion using hightech liquid to solid materials."
+	icon_state = "Kluysfluff1"
+	item_state = "Kluysfluff1"
+	blood_overlay_type = "coat"
+	body_parts_covered = UPPER_TORSO|ARMS
+
+	/obj/item/clothing/suit/kluysfluff/verb/toggle()
```

Comment:
This is technically /obj/item/clothing/suit/kluysfluff/obj/item/clothing/suit/kluysfluff/verb/toggle().

Also known as; Should be tabbed down one.